htlcswitch/switch.go

```diff
@@ -1068,12 +1071,12 @@ func (s *Switch) handlePacketForward(packet *htlcPacket) error {
 					}))
 			}
 
-			addErr := fmt.Errorf("incoming HTLC(%x) violated "+
+			log.Tracef("incoming HTLC(%x) violated "+
```

Comment:
Was this previously logged on trace?

Reply:
Since we've removed the additional error from `failAddPacket`, we are logging the `FailureDetail`'s string in `FailAddPacket`. Since these strings are static, we are no longer logging the payment hash and outgoing channel id, but we are logging the link error. 

I downgraded this to trace so that information is not lost entirely, but is not duplicate logged unless we are concerned with very low level logging.